pkg/apis/pipeline/v1alpha1/step_replacements.go

```diff
@@ -0,0 +1,67 @@
+/*
+ Copyright 2019 The Tekton Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
<soapbox> 
I'd actually like to see if we can move some of these things _out_ of this package, so that `v1alpha1` only contains structs, and no validation, merging, substitution, etc.

It's useful to have a low-dependency structs-only package for integrators to use only to send requests to a Tekton-enabled cluster, without having to import all the code Tekton uses to deal with those requests.

Future change though. 😄 
</soapbox>